apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -17,6 +17,10 @@ const title = computed(() => {
     return `Send <b>${_n(formatUnits(props.tx._form.amount, 0))}</b> NFT to <b>_NAME_</b>`;
   }
 
+  if (props.tx._type === 'stakeToken') {
+    return `Stake <b>${_n(formatUnits(props.tx.value, 18), 'standard', { formatDust: true })} ${'ETH'}</b> with <b>Lido</b>`;
```

Comment:
By losing the `token` information in the tx, we also lost the ability to easily identify the staked token.

We could bring that back by passing `contractAddress` and `networkId` to the `_form`, and rebuild it in this component, but seems ugly